app/models/pusher.rb

```diff
@@ -158,7 +158,15 @@ def notify_unauthorized
   def gem_security_policy
     @gem_security_policy ||= begin
       # Verify that the gem signatures match the certificate chain (if present)
-      policy = Gem::Security::LowSecurity.dup
+      policy = Gem::Security::Policy.new(
```

Comment:
Can we move this somewhere outside of this method into PushPolicy? :thinking: into model maybe?

Reply:
Good idea, I can define such a class under `app/models`.  I plan on also defining a PushPolicy in rubygems' `policies.rb` file, as suggested [here](https://github.com/rubygems/rubygems.org/issues/2830#issuecomment-951965495).  In the future, once rubygems.org moves to a rubygems having that constant, we can remove `app/models/push_policy.rb`.

What do you think?